File: 2023/java/src/test/java/org/frieder/aoc/day6/TestDay6A.java
package org.frieder.aoc.day6;

import org.frieder.aoc.day6.a.Solution6A;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay6A {
    @Test
    public void testExample() throws IOException {
        assertEquals(288, Solution6A.getResult("../resources/6/test_a.txt"), 0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(800280, Solution6A.getResult("../resources/6/input.txt"), 0);
    }
}

File: 2023/php/test/TestDay6A.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day6\a\Solution6A;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay6A extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(288, Solution6A::getResult("../resources/6/test_a.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(800280, Solution6A::getResult("../resources/6/input.txt", TestLogger::createTestLogger()));
    }

}